src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/DbBatchCommandCollection.cs

```diff
@@ -5,7 +5,7 @@
 using System.Collections;
 using System.Collections.Generic;
 
-namespace System.Data.Common
+namespace Microsoft.Data.Common
```

Comment:
I'd advise keeping the exact same namespace as the built-in type this is copying - any particular reason to not do this?

Reply:
Yes. It doesn't load because the type already exists in another assembly, from the system. The same error as before. 

Users can't see these types through the ref assembly they're only there because they're part of the implementation and we want to be able to share code. In theory if they developed directly against the implementation assembly or used private reflection they could see them but those aren't supported scenarios.